### PR TITLE
IncludeFixer - Add missing test case

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by gharlan** [comment]
:+1:

File: Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php [hunk 1]
```diff
@@ -32,6 +32,11 @@ public function testFixProvider()
         $template = '<?php %s';
         $tests = array();
         foreach (array('require', 'require_once', 'include', 'include_once') as $statement) {
+            $test[] = array(
+                sprintf('<?php %s "foo.php" ?>', $statement),
+                sprintf('<?php %s("foo.php") ?>', $statement),
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: It duplicates this:
https://github.com/SpacePossum/PHP-CS-Fixer/blob/1_12_feature_IncludeFixer_add_missing_test_case/Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php#L67

Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: No it doesn't? This test specifically doesn't have a semicolon.

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: This only proves how cool is to not have any description in PR/issue

File: Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php [hunk 2]
```diff
@@ -32,6 +32,11 @@ public function testFixProvider()
         $template = '<?php %s';
         $tests = array();
         foreach (array('require', 'require_once', 'include', 'include_once') as $statement) {
+            $test[] = array(
+                sprintf('<?php %s "foo.php" ?>', $statement),
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: Other cases use `$template`, this one not.

File: Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php (hunk 1 above)
Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: I assume this PR came out of this convesation: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1662#discussion_r49275979.

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: Sorry @keradus for not adding a description as I should have, it is indeed after the discussion as stated by @GrahamCampbell.

File: Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php (hunk 2 above)
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: updated to use the template, also added a test with comment tokens.

**Comment by keradus** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**